src/app/pages/AccountPage/Features/TransactionHistory/index.tsx

```diff
@@ -19,13 +23,17 @@ interface Props {}
 export function TransactionHistory(props: Props) {
   const { t } = useTranslation()
   const allTransactions = useSelector(selectTransactions)
+  const transactionsError = useSelector(selectTransactionsError)
   const address = useSelector(selectAccountAddress)
   const transactionComponents = allTransactions.map((t, i) => (
     <Transaction key={i} transaction={t} referenceAddress={address} />
   ))
 
   return (
     <Box gap="small" margin="none">
+      {transactionsError ? (
+        <p>Couldn't load transactions. List may be empty or out of date. {transactionsError}</p>
```

Comment:
I think this error should be translated using `t()`?

Reply:
added